overalls.go

```diff
@@ -67,6 +73,7 @@ var (
 	modeRegex       = regexp.MustCompile("mode: [a-z]+\n")
 	srcPath         string
 	projectPath     string
+	goBinary        string
```

Comment:
Not sure about the alphabetization here, so I picked somewhere in the middle? Happy to move if there's a pattern I missed